universe/ValueRefs.cpp

```diff
@@ -1613,6 +1674,26 @@ int ComplexVariable<int>::Eval(const ScriptingContext& context) const
         }
         return 0;
     }
+    else if (variable_name == "Named") {
+        if (!m_string_ref1)
+            return 0;
+        std::string rule_name = m_string_ref1->Eval();
+        if (rule_name.empty()) {
+            ErrorLogger() << "Named ComplexVariable<int> was given an empty name instead of a registered value ref name";
+            return 0;
+        }
+        if (!m_int_ref1) {
```

Comment:
some explanation of what's going on here needed... if it's actually needed and kept...

Reply:
Removed "Named" complex value ref completely